Gemfile

```diff
@@ -55,6 +55,10 @@ gem 'dynflow', '>= 0.8.29', '< 1.0.0'
 gem 'daemons'
 gem 'get_process_mem'
 
+# telemetry
+gem 'prometheus-client'
```

Comment:
Shouldn't this dependencies be optional?

Reply:
Do you mean separate bundler groups for both prometheus and statsd? It's part of core settings.yaml there is no precedent. Plugins have usually only Administer Settings, but this needs to be setup early in the boot so it is actually available for all plugins. I'd rather waive this into core dependencies. Or explain how do you think this should be done in more detail.

Reply:
Moved to own bundler group, added a note in YAML settings that this must be installed prior enabling telemetry.